index.js

```diff
@@ -20,11 +23,11 @@ LESSCompiler.prototype.compile = function(params, callback) {
   var data = params.data;
   var path = params.path;
 
-  less.render(data, {
+  var options = _.extend({}, this.options, {
```

Comment:
Just iterate over the keys the old-fashioned way. No need to add underscore for this.